README.md

```diff
@@ -64,3 +64,86 @@ consumer = Consumer(client_id='fill me in',
+                    client_id='fill me in',
+                    client_secret='fill me in',
+                    domain='gcn.nasa.gov')
+
+topics = ['gcn.classic.voevent.FERMI_GBM_SUBTHRESH']
+consumer.subscribe(topics)
+
+while True:
+    for message in consumer.consume():
+        print(message.value())
+        consumer.commit(message)
+```
+
+**How can I read messages beginning at the earliest available messages for a given stream?**
+
+You can begin reading a given topic stream from the earliest message that is present in the stream buffer by setting the Group ID to an empty string and applying the ‘earliest’ setting for the auto offset reset option in the configuration dictionary argument for the Consumer class. This feature allows the user to scan for older messages for testing purposes or to recover messages that may have been missed due to a crash or network outage. Just keep in mind that the stream buffers are finite in size. They currently hold messages from the past few days.
+
+Example code:
+```python3
+from gcn_kafka import Consumer
+
+config = {'group.id': '',
+          'auto.offset.reset': 'earliest'}
+
+consumer = Consumer(config=config,
+                    client_id='fill me in',
+                    client_secret='fill me in',
+                    domain='gcn.nasa.gov')
+
+topics = ['gcn.classic.voevent.INTEGRAL_SPIACS']
+consumer.subscribe(topics)
+
+while True:
+    for message in consumer.consume():
+        print(message.value())
+```
+
+Note: Adding a timeout argument, given as an integer number of seconds, to consume() will allow the program to exit quickly once it has reached the end of the existing message buffer. This is useful for users who just want to recover an older message from the stream.
+
+**How can I search for messages occurring within a given date range?**
+
+To search for messages in a given date range, you can use the offsets_for_times() function from the Consumer class to get the message offsets for the desired date range. You can then assign the starting offset to the Consumer and read the desired number of messages. When doing so, keep in mind that the stream buffers are finite in size. It is not possible to recover messages prior to the start of the stream buffer. The GCN stream buffers are currently set to hold messages from the past few days.
+
+Example code:
+```python3
+import datetime
+from gcn_kafka import Consumer
+from confluent_kafka import TopicPartition
+
+consumer = Consumer(client_id='fill me in',
+                    client_secret='fill me in',
+                    domain='gcn.nasa.gov')
+
+# get messages occurring 3 days ago
+timestamp1 = int((datetime.datetime.now() - datetime.timedelta(days=3)).timestamp() * 1000)
+timestamp2 = timestamp1 + 86400000 # +1 day
+
+topic = 'gcn.classic.voevent.INTEGRAL_SPIACS'
+start = consumer.offsets_for_times(
+    [TopicPartition(topic, 0, timestamp1)])
+end = consumer.offsets_for_times(
+    [TopicPartition(topic, 0, timestamp2)])
```

Comment:
Are we sure that `datetime.datetime.now()` uses the proper timezone? Hopefully the Kafka broker is using UTC offsets, but I don't know.

Reply:
So it looks like the timezone is irrelevant because the selections are performed based on the POSIX timestamp. The timezone is only applied when printing the time/date string. I can confirm that the Kafka broker returns UTC times that match the UTC trigger times in GBM Alerts when I use `datetime.datetime.utcfromtimestamp()` to convert to a time/date string in UTC. I think everything is sensibly setup.